### PR TITLE
📝 Add PteraSoftware to external gallery

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

@camUrban Thank you for a [great example](https://github.com/camUrban/PteraSoftware) of PyVista use. We really glad that PyVista mania is increasing.

 

ptera_software

### Details

- https://github.com/pyvista/pyvista/issues/2073

### CODE REVIEW

File: doc/make_external_gallery.py [hunk 1]
```diff
@@ -115,6 +115,10 @@ def format(self):
         description="Stéphane Laurent's blog showing a sample of the animations they realized with PyVista.",
         link="https://laustep.github.io/stlahblog/posts/MyPyVistaArtworks.html",
         image="Duoprism_3-30.gif"),
+    ptera_software=Example(title="PteraSoftware",
+        description="Ptera Software is a fast, easy-to-use, and open-source software package for analyzing flapping-wing flight.",
+        link="https://github.com/camurban/pterasoftware",
+        image=" ptera_software.gif"),
```
Comment (adeak) [inline doc/make_external_gallery.py:121]: This might cause a problem
```suggestion
        image="ptera_software.gif"),
```

**Comment by akaszynski** [comment]
@banesullivan, builds seem to be disabled. I'll look at the yml later today, but was this intended?

**Comment by banesullivan** [comment]
Not intended... I'll debug it. I opened and closed some prs yesterday where I was trying to debug it. 

Almost seems like the no-ci branch check is flaky

**Comment by banesullivan** [comment]
Please rebase as I reverted the no-ci changes

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks @tkoyama010!

**Comment by tkoyama010** [comment]
Thanks for review.

**Pull request merged**